clients/reality-data/src/RealityDataClient.ts

```diff
@@ -84,7 +91,7 @@ export class RealityData extends WsgInstance {
   public streamed?: boolean;
 
   @ECJsonTypeMap.propertyToJson("wsg", "properties.Type")
-  public type?: string;
+  public type?: RealityDataType;
```

Comment:
The reality data types are strings because the underlying REST API defines types as strings. The type property is open in the sense there are reserved keywords that define specific reality data types (See SWAGGER page of the service) but user can introduce new Reality Data types as needed for their own use or for some reality data type designed by them. 
If we want to restrict the current client to access only reality data types that are supported by the various display component then it is acceptable to limit the type to those types only. If, however, we want to preserve the ability to use other reality data types (either reserved or user-introduced) then the type property should be maintained a string.

Reply:
Thanks for pointing this out. I went ahead and put it back to a string.